LEAF_Request_Portal/sources/System.php

```diff
@@ -122,9 +122,7 @@ public function updateService(int $serviceID): array
                             if ($backups['status']['code'] == 2) {
                                 // check if this service is also an ELT
                                 // if so, update groups table
-                                $tagged = $tag->groupIsTagged($serviceID, Config::$orgchartImportTags[0]);
-
-                                if ($serviceID == $quadID && $tagged['status']['code'] == 2 && !empty($tagged['data'])) {
+                                if ($serviceID == $quadID) {
```

Comment:
What is this status code stuff? I see we are checking a bunch of different variables with this status code and we are not checking if this tagged is empty. I may be missing something obvious why this was needed or not needed.